src/v2/Apps/BuyerGuarantee/Routes/BuyerGuaranteeIndex.tsx

```diff
@@ -460,6 +460,7 @@ export const BuyerGuaranteeIndex: React.FC<BuyerGuaranteeIndexProps> = ({
             backgroundColor={color("black5")}
             gridTemplateColumns="repeat(3, 1fr)"
             mt={2}
+            px={2}
```

Comment:
@ashleyjelks - Apologize, but not sure I follow your comment on padding for mobile/xs. Is that not addressed here? Realized I forgot to include the pic, though! Let me know if I'm just missing something obvious!

<img width="193" alt="Screen Shot 2021-05-26 at 2 25 42 PM" src="https://user-images.githubusercontent.com/38149304/119712163-517f4380-be2e-11eb-8756-63b6cb956a31.png">